internal/pkg/manifest/backend_svc.go

```diff
@@ -28,10 +28,10 @@ type BackendServiceConfig struct {
 	ImageConfig      ImageWithPortAndHealthcheck `yaml:"image,flow"`
 	ImageOverride    `yaml:",inline"`
 	TaskConfig       `yaml:",inline"`
-	*Logging         `yaml:"logging,flow"`
+	Logging          `yaml:"logging,flow"`
 	Sidecars         map[string]*SidecarConfig `yaml:"sidecars"`
```

Comment:
can you remind me why we keep the pointer if it's a `map[string]<PStruct>`? are there other scenarios where the pointer is kept?

Reply:
so that `mergo` will deep merge `SideCarConfig` as well. For example, say the original has a key of `sidecar1`, and the override also has a key `sidecar1`. Without the pointer, `mergo` replaces the `sidecar1` in the original with the one in  override, instead of deep merging them.

Reply:
Nice can we add this as a comment if it's not mentioned somewhere 🥺